tenv.go

```diff
@@ -24,13 +27,27 @@ var Analyzer = &analysis.Analyzer{
 var (
 	A     = "all"
 	aflag bool
+
+	Go     = "go"
+	goflag string
 )
 
 func init() {
 	Analyzer.Flags.BoolVar(&aflag, A, false, "the all option will run against all method in test file")
+	Analyzer.Flags.StringVar(&goflag, Go, "1.17", "Go version")
 }
 
 func run(pass *analysis.Pass) (interface{}, error) {
+	lower, err := goVersionLower117(goflag)
+	if err != nil {
+		return nil, err
+	}
+
+	if lower {
+		// Do nothing because T.Setenv added in go1.17
```

Comment:
I think it's better to print the warning about specifying lower or empty version. What do you think ?